website/static/js/pages/profile-settings-addons-page.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+require('../../css/user-addon-settings.css');
```

Comment:
Since we have `resolve.root` set to `website/static` in our webpack config, you should be able to do `require('css/user-addon-settings.css')`.